pkg/config/remote/service/service.go

```diff
@@ -847,6 +872,7 @@ func (s *CoreAgentService) ClientGetConfigs(_ context.Context, request *pbgo.Cli
 		Targets:       canonicalTargets,
 		TargetFiles:   targetFiles,
 		ClientConfigs: matchedClientConfigs,
+		ConfigStatus:  pbgo.ConfigStatus_CONFIG_STATUS_EXPIRED,
```

Comment:
I think you want OK here and thus probably some assertions in tests to make sure that's the case.